pkg/server/server.go

```diff
@@ -36,10 +39,36 @@ func (s *Server) Start() {
 	}
 }
 
-func (s *Server) createRouters() *mux.Router {
-	r := mux.NewRouter()
-	r.HandleFunc("/healthcheck", s.healthcheckHandler)
-	return r
+func (s *Server) checkDeviceIdPermission() negroni.Handler {
+	return negroni.HandlerFunc(func(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+
+		url := strings.Split(r.URL.String(), "?")
+
+		params := strings.Split(url[0], "/")
+
+		if r.Method == "POST" || params[2] == "teste-12345" {
+			next(w, r)
+		} else {
+			w.WriteHeader(http.StatusUnauthorized)
+		}
+	})
```

Comment:
Not sure if this is the best way to do this :thinking: 
Any thoughts? @esdrasjnr @joaoaneto

Reply:
@lcbm @Igoraamc I think it's better to define the function outside the context of `negroni.HandlerFunc` call. This will make it simple to be read, IMO :smiley:

Reply:
I agree with both of you. But this is a temporary code, and it will be all replaced when the authorization feature is implemented. However, if you guys think that this code shouldn't be here, I can leave this function blank.

Reply:
I think it'd better use this authentication middleware only when we need it, doing the authentication process completely, which is a task someone will work on. For now, we can do all the operations without authentication.